deployment/postgres-init-db/sql/tables/merlin/external_events/derivation_group.sql

```diff
@@ -0,0 +1,18 @@
+create table merlin.derivation_group (
```

Comment:
Can a derivation group only contain a single source_type?

Reply:
Updated comment in [clarify derivation_group.sql](https://github.com/NASA-AMMOS/aerie/commit/1bba56fde319c72624e23af8a6afc4043ec8995f).